imagetest/test_suites/metadata/shutdown_script_test.go

```diff
@@ -1,58 +0,0 @@
-// +build cit
```

Comment:
why deleted?

Reply:
startup_script_test.go and shutdown_script_test.go were combined into startup_shutdown_script_test.go since most of the tests that were previously separate use the same logic for testing both the startup and shutdown scripts.